docs/testing/playwright/02-e2e-testing.md

```diff
@@ -61,6 +66,9 @@ test.describe('my-component', () => {
 });
 ```
 
+In the above snippet, the path string passed to `page.goto()` should be the _absolute_ path to the HTML file from the
+[dev server's served directory](../../config/dev-server.md#root).
```

Comment:
I'm still confused where this `my-component.e2e.html` should live - if my `root` isn't set (it has a default level of `www`), how does `my-component.e2e.html` get put in `www`?